src/app/workflow/workflow.component.html

```diff
@@ -170,16 +170,25 @@ <h3>
           </mat-tab>
           <div *ngIf="entryType === EntryType.BioWorkflow">
             <mat-tab id="launchTab" label="Launch">
-              <app-launch
-                *ngIf="workflow?.workflowVersions.length > 0; else noVersions"
-                [selectedVersion]="selectedVersion"
-                [basePath]="workflow?.path"
-                [path]="workflow?.full_workflow_path"
-                [canRead]="canRead"
-                [canWrite]="canWrite"
-                [isOwner]="isOwner"
-                [mode]="workflow?.mode"
-              ></app-launch>
+              <div *ngIf="!(launchSupport$ | async); else launchSupported">
+                <div class="p-3">
+                  <mat-card class="alert alert-warning" role="alert">
+                    <mat-icon>warning</mat-icon> Dockstore does not yet have customized launch-with instructions for this language.
```

Comment:
As in Denis' suggestion, I think it should mention the specific language rather than `this language`. Or is that going to be hard to do generically because we say `gxformat2` instead of Galaxy?

Reply:
I think mentioning the language itself adds little to no value to the user.  And yes, the backend provides "gxformat2", Nextflow provides "NFL".  All of which we have and would have overrided in the frontend

Reply:
Looking at TRS again in a coincidence, I'm tempted to switch it to GALAXY anyway. 
For example, we don't treat WDL draft-2 differently from WDL version 1